bigbluebutton-html5/imports/ui/components/common/menu/component.jsx

```
@@ -156,10 +158,34 @@ class BBBMenu extends React.Component {
+                    {a.iconRight ? <Styled.IconRight color={textColor} iconName={a.iconRight} key="iconRight" /> : null}
+                    {(isTitle && titleActions?.length > 0) ? (
+                      titleActions.map((item) => (
+                        <Styled.TitleAction
+                          tooltipplacement="right"
+                          size="md"
+                          onClick={item.onClick}
+                          circle
+                          tooltipLabel={item.tooltip}
+                          hideLabel
+                          icon={item.icon}
+                        />
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)